### ISSUE TITLE
API Completion stream abort broken

### ISSUE THREAD

**Issue by lufixSch** [oobabooga/text-generation-webui#5871]
### Describe the bug

Since I pulled the last snapshot (I think) stopping/aborting a completion stream (from the API) is weirdly broken.
With llama.cpp it just stopped working completely. Even though the stream is aborted/stopped the generation continues. 
With exllamav2 the completion stops but afterwards I am unable to generate any other text either using the API or the Gradio GUI

When using the GUI everything works fine. I'm unsure if its a problem in the code or related to package changes, because I tried going back to previous snapshots up to `snapshot-2024-03-24` and the behavior did not change. Unfortunately I didn't have time to also try reinstalling dependencies to match the snapshot (AMD always takes a bit longer xD).
It is also possible the problem is on my side but I have no Idea what could cause that. I even tried different applications which use the API to check if its caused on the application side.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

1. Run WebUI with API
2. Load model with llama.cpp (1) or exllama (2)
3. Trigger text generation using the API
4. Stop the text generation by aborting the stream
5. (2) Trigger text generation using the API or GUI

### Logs

```shell
<none>
```

### System Info

```shell
Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.11
KDE Frameworks Version: 5.115.0
Qt Version: 5.15.12
Kernel Version: 6.6.26-1-MANJARO (64-bit)
Graphics Platform: X11
Processors: 20 × 13th Gen Intel® Core™ i5-13500
Memory: 31.1 GiB of RAM
Graphics Processor: AMD Radeon RX 7900 XTX
Manufacturer: Micro-Star International Co., Ltd.
Product Name: MS-7D98
System Version: 1.0
```

**Comment by p-e-w** [comment]
Fixed in #5873.

**Comment by lufixSch** [comment]
Thanks! I'll try downgrading `sse-starlette`
